pyproject.toml

```diff
@@ -38,7 +38,7 @@ influxdb = "^5.3.1"
 # required for tests to pass right now. There are often issues when pinning to a branch
 # and calling poetry update. I have had to completely remove my venv and reinstall in order
 # to get the latest updates.
-alfalfa-client = { git = "https://github.com/nrel/alfalfa-client.git", branch = "upload_model_check" }
+alfalfa-client = { git = "https://github.com/nrel/alfalfa-client.git", branch = "develop" }
```

Comment:
do you mean with a release of alfalfa_client?
everytime an API change is made the client has to change, so usually I wait until it's time to release the new version of alfalfa to release the new client so that people's setups don't break by automatically installing incompatible versions of the alfalfa client.

Additionally, this package inclusion is only used for testing so it doesn't really affect anyone besides us. The reason it was set to the `upload_model_check` branch instead of dev for a while was because there were changes made in the client to fix a breaking bug that hadn't been merged yet.